google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -115,3 +115,30 @@
 """
 
 # TODO: Add a RAW_GROUP_MEMBERS_TABLE.
+
+CREATE_BUCKETS_TABLE = """
+    CREATE TABLE `{0}` (
```

Comment:
I think that other interesting fields like acl[], or defaultObjectAcl will be added later?  If so, can we please add a TODO here (or somewhere) to remember?